frontend/src/pages/index.vue

```diff
@@ -66,31 +66,31 @@ if (userLibraries.isReady) {
 const homeSections = computed<HomeSection[]>(() => {
   // Filter for valid sections in Jellyfin Vue
   // TODO: Implement custom section order
-  let homeSectionsArray = pickBy(
+  const homeSectionPrefs = pickBy(
```

Comment:
Previously homeSectionsArray was untyped and started off as an object, then got re-assigned to an array later, so it's type was actually changing. The logic is identical, but I think it's just easier to follow now.